### DISCUSSION TITLE
Data  entry on the panel

### DISCUSSION THREAD

**Discussion by rudgej** [Blackymas/NSPanel_HA_Blueprint#1192] [Q&A]
I have a start time for my morning automations. 

Is there a way that this time can be edited on the panel?

I have added the helper and tried a long push but nothing happens. 

I am not sure how I could display its value, but if I could I could then have a button to increment up / down the time by 10 mins each push which would achieve the same thing by running an automation.

Advise please!

**Comment by edwardtfn** [comment]
- #1734

**Reply by rudgej** [reply]
thank you
